README.md

```diff
@@ -249,8 +249,10 @@ Usage of gcp-exporter:
     	Disables the metrics collector for the Cloud Functions
   -collector.iam.disable
     	Disables the metrics collector for the Cloud IAM
+  -collector.kubernetes.config string
+    	Specifies specific settings for the GKE collector (default "{\"enableClusterAndNodePoolInfoMetric\": false}")
   -collector.kubernetes.disable
-    	Disables the metrics collector for the Kubernetes (GKE)
+    	Disables the metrics collector for Google Cloud Engine
```

Comment:
It's called [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine) or "GKE" **not** Google Cloud Engine